utils/send_slack_message.py

```diff
@@ -3,13 +3,13 @@
 # Author: M. Reinmuth, B. Herfort
 ####################################################################################################
 import json
-
 from slackclient import SlackClient
 
+CONFIG_PATH = './cfg/config.cfg'
```

Comment:
So actually I think this invites for a config file that loads the json from the CONFIG_PATH and exposes-it, and this file would import the config.py instead. See here as an example - https://github.com/BeaconLabs/encompass/blob/master/backend/config/config.py

Without getting into a file with that many functions, I think that at least loading the config in a config.py makes sense.